.github/workflows/build-and-release.yml

```
@@ -353,3 +353,63 @@ jobs:
+          dpkg-deb -x pictopy.deb extract/
+          
+          # Create tar.gz with proper structure
+          cd extract
+          tar -czvf "../pictopy_${VERSION}_amd64.tar.gz" .
+          cd ..
+          
+          # Calculate checksum
+          sha256sum "pictopy_${VERSION}_amd64.tar.gz" > "pictopy_${VERSION}_amd64.tar.gz.sha256"
+
+      - name: Upload tar.gz to release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.9)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)